tox.ini

```diff
@@ -96,7 +97,16 @@ commands =
 max_line_length = 88
 exclude =
     docs/src/examples/
+docstring-convention = numpy
 per-file-ignores =
     # D205 and D400 are incompatible with the requirements of sphinx-gallery
     examples/**:D205, D400
-extend-ignore = E203
+ignore = 
+    E203
+    D100
+    D101
+    D102
+    D205
+    D400
+    D401
+    W503
```

Comment:
I just assume this makes sense

Reply:
I mean a lot of them are very [useless](http://www.pydocstyle.org/en/2.1.1/error_codes.html). You can take a look if you disagree with the ones I disabled.

The ones I disabled mean

D100	Missing docstring in public module
D101	Missing docstring in public class
D102	Missing docstring in public method
D205	1 blank line required between summary line and description

D400	First line should end with a period
D401	First line should be in imperative mood

Every Code that we remove from this list will require further changes to the docstrings.

E203 and W503 have to be disabled due to incompatibilities with black.